cli/create/templates/cartridge/README.md

```diff
@@ -0,0 +1,78 @@
+# Simple Tarantool Cartridge-based application
+
+This a simplest application based on Tarantool Cartridge.
+
+## Quick start
+
+To build application and setup topology:
+
+```bash
+cartridge build
+cartridge start -d
+cartridge replicasets setup --bootstrap-vshard
+```
+
+Now you can visit http://localhost:8081 and see your application's Admin Web UI.
+
+**Note**, that application stateboard is always started by default.
+See [`.cartridge.yml`](./.cartridge.yml) file to change this behavior.
+
+## Application
+
+Application entry point is [`init.lua`](./init.lua) file.
+It configures Cartridge, initializes admin functions and exposes metrics endpoints.
+Before requiring `cartridge` module `package_compat.cfg()` is called.
+It configures package search path to correctly start application on production
+(e.g. using `systemd`).
+
+## Roles
+
+Application has one simple role, [`app.roles.custom`](./app/roles/custom.lua).
+It exposes `/hello` and `/metrics` endpoints:
+
+```bash
+curl localhost:8081/hello
+curl localhost:8081/metrics
+```
+
+Also, Cartridge roles [are registered](./init.lua)
+(`vshard-storage`, `vshard-router` and `metrics`).
+
+You can add your own role, but don't forget to register in using
+`cartridge.cfg` call.
+
+## Instances configuration
+
+Configuration of instances that can be used to start application
+locally is places in [instances.yml](./instances.yml).
+It is used by `cartridge start`.
+
+## Topology configuration
+
+Topology configuration is described in [`replicasets.yml`](./replicasets.yml).
+It is used by `cartridge replicasets setup`.
+
+## Tests
+
+Simple tests are placed in [`test`](./test) directory.
+To run tests, say
+
+```bash
+cartridge stop  # to prevent "address already in use" error
+./deps.sh
+.rocks/bin/luacheck .
```

Comment:
But it's not a tests, it's a lint...

Reply:
I've changed this section structure - installing dependencies and linter are described separately.